src/connector.js

```diff
@@ -0,0 +1,80 @@
+const winston = require('winston');
+const { ConnectionStringParser } = require('connection-string-parser')
+const { ReplSet, Server, MongoClient } = require('mongodb');
+
+const connectionStringParser = new ConnectionStringParser({
+  scheme: "mongodb",
+  hosts: []
+});
+
+const connector = {
+  client: null,
+
+  connected: false,
+
+  connectionString: '',
+
+  connect: async (connectionString) => {
+    try {
+      const dbClient = await MongoClient.connect(connectionString, {
+        poolSize: 200,
+        useNewUrlParser: true,
+      });
+
+      this.client = dbClient;
+      this.connected = true;
+      this.connectionString = connectionString;
+
+      return dbClient
+    } catch (error) {
+      winston.log('error', error)
+    }
+  },
+
+  dbConnect: (appId) => {
+    if (this.connected) {
+      return this.client.db(appId)
+    }
+
+    throw new Error('Database not connected');
+  },
+
+  replSet(configs) {
```

Comment:
merge this function with connect - connect should be able to connect to replSet MognoDB or standalone MongoDB;s.

Reply:
and remove this function,

Reply:
This method is used by the `create` method in `src/utils`. I'm not sure how to remove it. Please advise.